### PR TITLE
Feat/notification redirect functionality

### PR DESCRIPTION
**Pull request by aliassan**
# Description

Make user redirect to teams page when they click on invitation.

## Type of change

- [ ] New feature (non-breaking change which adds functionality)

# Checklist:

* **Please check if the PR fulfills these requirements**

- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added / updated (for bug fixes / features)

### CODE REVIEW

**Review by hermannleboss — APPROVED** [review]
LGTM 

Fix the test errors

**Review by dasarathg68 — APPROVED** [review]
LGTM

**Comment by dasarathg68** [comment]
@aliassan suggestion: use prettier vscode plugin for code formatting.

File: app/src/components/NotificationDropdown.vue [hunk 1]
```diff
@@ -75,18 +76,32 @@ const {
...
+}
+
+const updateNotification = async (notification: Notification) => {
+  updateEndPoint.value = `notification/${notification.id}`
 
   await executeUpdateNotifications()
   await executeFetchNotifications()
+
+  if (isInvitation(notification) && notification.resource) {
+    router.push(`/${notification.resource}`)
```
Comment (dasarathg68) [inline app/src/components/NotificationDropdown.vue:103]: The test errors have got to do something with the router being called from the components (MetaMask provider is somehow not accessible from the component level I think). 
Comment this, and you'll be able to see that the tests pass.

Comment (aliassan) [reply app/src/components/NotificationDropdown.vue:103]: The tests pass but I lose the functionality

Comment (dasarathg68) [reply app/src/components/NotificationDropdown.vue:103]: Yeah I get that, I was just pointing the source of the error.

**Review by dasarathg68 — APPROVED** [review]
LGTM

**Pull request merged**